app/src/main/resources/application.properties

```diff
@@ -4,6 +4,9 @@ integration.intygproxyservice.person.endpoint=/api/v1/person
 integration.intygstjanst.scheme=http
 integration.intygstjanst.certificate.endpoint=/inera-certificate/internalapi/citizen
 
+integration.webcert.scheme=http
+integration.webcert.certificate.endpoint=/
```

Comment:
Which end point will we use?

Reply:
integration.webcert.scheme=http
integration.webcert.certificate.endpoint=/internalapi/certificate/{certificateId}